src/network/postgres/postgres_command_factory.cpp

```diff
@@ -2,7 +2,7 @@
 #include <memory>
 namespace terrier::network {
 
-std::shared_ptr<PostgresNetworkCommand> PostgresCommandFactory::PacketToCommand(InputPacket *packet) {
+PostgresNetworkCommand *PostgresCommandFactory::PacketToCommand(InputPacket *packet) {
```

Comment:
Any reason this shouldn't be a ManagedPointer that's being returned?

Reply:
In theory it would be a unique_ptr, if anything, since ownership belongs to the caller. I'll see if I can make that happen with the casting stuff since unique_ptr doesn't have the *_pointer_cast methods that shared_ptr does.